src/sentry/demo/apps.py

```diff
@@ -0,0 +1,11 @@
+from django.apps import AppConfig
+
+
+from .models import DemoOrganization, DemoUser
+
+
+class Config(AppConfig):
+    name = "sentry.demo"
```

Comment:
I thought I ran into a problem with generating migrations for apps with a `.` in the name, but looks like it ran fine?